bats/tests/helpers/vm.bash

```diff
@@ -43,7 +43,7 @@ start_container_engine() {
     mkdir -p "$PATH_CONFIG"
     cat <<EOF >"$PATH_CONFIG_FILE"
 {
-  "version": 6,
+  "version": 7,
```

Comment:
Just for my understanding: this change isn't really necessary, right? The settings API would simply migrate the version 6 settings to version 7 internally, and it would still work?

If this is true, then in the future we don't need to bump versions right away. Leaving it at the previous version has the advantage the test suite would still work with a slightly older dev/CI build from the same release cycle.

Reply:
Just running into this: can't run this test with the 1.9 Tech Preview anymore:

```
   errors in attempt to update settings:
   updating settings requires specifying version = 6, but received version 7
   Changing field version via the API isn't supported.
```

From the error message it sounds like we don't support version migration in the API, but of course in this case it can't work anyways. I wonder if it would have worked the other way around. Given that we have migrations for `settings.json` I wonder if this should work for the API as well.